src/soundsourceoggvorbis.cpp

```diff
@@ -51,9 +51,11 @@ inline int getByteOrder() {
  */
 
 SoundSourceOggVorbis::SoundSourceOggVorbis(QString qFilename)
-: Mixxx::SoundSource(qFilename)
+: Mixxx::SoundSource(qFilename),
```

Comment:
check out some other files, I think the : should be 4 or 8 spaces in?  I just let Eclipse format it so I can't remember.